### DISCUSSION TITLE
Yet Another Activity Idea: Arousal reducing activities

### DISCUSSION THREAD

**Discussion by anchillabc** [littlesera/LSCG#394] [Ideas]
The idea is that while there are lots of activities which increase arousal, there aren't any that reduce it.

So, I came up with a few activities designed to actively reduce arousal
- Meditate
- Take deep breaths

To keep these from being abused, the effectiveness of the second action can be tied to gag level - the higher, the less effect it would have, with no effect at all on active choking levels (regardless of breathplay settings)

For meditation, the effect might be slower over a longer period of time, without stacking, that should take care of abuse.

**Comment by littlesera** [comment]
bump
